py/picca/io.py

```diff
@@ -346,11 +531,19 @@ def read_data(in_dir,
     """
     userprint("mode: " + mode)
     # read quasar characteristics from DRQ catalogue
-    ra, dec, z_qso, thingid, plate, mjd, fiberid = read_drq(drq_filename,
-                                                            z_min,
-                                                            z_max,
-                                                            keep_bal,
-                                                            bi_max=bi_max)
+
+    try:
```

Comment:
given that function `read_data` has de `mode` variable, wouldn't it be better to solve this with an if/else block instead of with a try/except?

Reply:
this particular case has been removed, similar try/excepts have been changed to more clear if/else conditions